api/src/services/postgis-service.ts

```diff
@@ -0,0 +1,26 @@
+import { Geometry } from 'geojson';
+import { IDBConnection } from '../database/db';
+import { PostgisRepository } from '../repositories/postgis-repository';
+import { DBService } from './db-service';
+import { Srid } from './geo-service';
+
+export class PostgisService extends DBService {
+  postgisRepository: PostgisRepository;
+
+  constructor(connection: IDBConnection) {
+    super(connection);
+    this.postgisRepository = new PostgisRepository(connection);
+  }
+
+  /**
+   * Convert the provided geometry into Well-Known Text (WKT) in the provided Spatial Reference ID (SRID).
```

Comment:
would be great if there was a link explaining what WKT is in this context

Reply:
Added.